src/ProtocolFees.sol

```diff
@@ -56,7 +56,7 @@ abstract contract ProtocolFees is IProtocolFees, PausableRole {
             if (!_success || _data.length > 32) return (false, 0);
 
             uint256 returnData;
-            assembly {
+            assembly ("memory-safe") {
```

Comment:
Accessing memory allocated by Solidity is memory safe i.e. data is allocated at line 52